nft-simple/test-contracts/exquisite-corpse/src/lib.rs

```diff
@@ -0,0 +1,345 @@
+/*!
+Some hypothetical DeFi contract that will do smart things with the transferred tokens
+*/
```

Comment:
This is too large for a test or example. We don't need a working app for a test.

Reply:
simplified it significantly